cadasta/api/test/test_login.py

```diff
@@ -4,32 +4,31 @@
 
 import unittest
 
+import os
 import qgis
 import logging
 from cadasta.api.login import Login
-from cadasta.test.utilities import get_qgis_app
 from qgis.PyQt.QtCore import QCoreApplication
 
-
-QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
-
-LOGGER = logging.getLogger('CadastaQGISPlugin')
+if not os.environ.get('ON_TRAVIS', False):
+    from cadasta.test.utilities import get_qgis_app
+    QGIS_APP = get_qgis_app()
```

Comment:
If I run the test from test_suite.test_package it will working fine even though I didn't use get_qgis_app